Sources/TuistCore/Extensions/SPMUtility+Extras.swift

```diff
@@ -0,0 +1,17 @@
+import SPMUtility
+
+extension Version {
+    /// Create a version object from string.
+    /// It does not have to be formatted to SPM's standards (i.e. minor and patch versions can be omitted if zero)
+    /// - Parameters:
+    ///   - unformattedString: The string to parse.
+    init?(unformattedString: String) {
```

Comment:
This constructor is a good candidate for a unit test. I'd test the following scenarios:

- "x.y.z"
- "x.y"
- "x"

Reply:
They are already there: `SPMUtility+VersionTests`

Reply:
Awesome! Ignore my comment then